core/drivers/crypto/se050/core/rsa.c

```diff
@@ -15,6 +15,10 @@
 #include <tee/tee_cryp_utl.h>
 #include <tee_api_defines_extensions.h>
 
+#define TEE_OAEP_ALG(__hash_alg) \
```

Comment:
What this macro does with a few words.

Reply:
from an OAEP hash algorithm, generates the OAEP algorithm 
(just undo what the crypto API did)

Reply:
I was more looking for a comment like:
/* Makes a TEE_ALG_RSAES_PKCS1_OAEP_MGF1_* from input hash function */